source/scripts/notesEditor.js

```diff
@@ -89,12 +91,14 @@ function addEditToggle(preview, id){
  */
 function addNewNoteButtons(id, db) {
     // create a save button
-    let saveButton = document.createElement('button');
-    saveButton.setAttribute('class', 'button-button');
-    saveButton.innerHTML = 'Save';
-    let buttonSection = document.querySelector('#option-button');
-    buttonSection.appendChild(saveButton);
-
+    let saveButton =document.querySelector('.save-button')
+    if(!saveButton){
```

Comment:
Don't we only call addNewNoteButtons() when there's no save button existent on the screen? We shouldn't have to check saveButton because we know it will be null.